docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -185,6 +206,24 @@ type ExpoAssetHeaderDictionary = {
 
 - `assetRequestHeaders`: MAY contain a dictionary of header (key, value) pairs to include with asset requests. Key and value MUST both be strings.
 
+### Directive Body
+
+Defined as JSON conforming to both the following `Directive` definition expressed in [TypeScript](https://www.typescriptlang.org/) and the detailed descriptions for each field:
+
+```typescript
+type Directive = {
+  type: string;
+  payload?: { [key: string]: any };
```

Comment:
The word "payload" to me emphasizes that the data is being transported and is the body of a message. That's true here but these fields are more like parameters and arguments to the directive.

Some parts of the HTML spec (like Content Security Policies) use the terms "directive name" and "directive value". I think "value" is too generic and also seems a bit off when some of our update directives don't even need payloads/values/parameters.

Four words I considered are:
- parameters: I think this is best and adds a little precision by implying they parameterize the directive while also staying fairly generic and flexible
- data: OK but rather generic
- value: too generic and also implies a bit that it's necessary
- arguments: while maybe it's more technically correct than parameters, it's less intuitive than parameters and also is a reserved word in JS and can make coding a little less smooth